vslib/SwitchState.cpp

```diff
@@ -199,6 +199,102 @@ void SwitchState::asyncOnLinkMsg(
     m_switchConfig->m_eventQueue->enqueue(std::make_shared<Event>(EVENT_TYPE_NET_LINK_MSG, payload));
 }
 
+sai_status_t SwitchState::getNetStat(
+        _In_ sai_stat_id_t counter_id, 
+        _In_ std::string& if_name, 
+        _Out_ uint64_t &counter)
+{
+    struct {
```

Comment:
please declare struct in separate file or in *.h file of SwithcState

Reply:
ok